ckanext/cfpb_extrafields/controllers/digimport.py

```diff
@@ -0,0 +1,47 @@
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from stringIO import StringIO
+import json
+import re
+import urllib
+
+from ckan.plugins.toolkit import BaseController, get_action, redirect_to, render, request, response, ValidationError
+
+from ckanext.cfpb_extrafields.digutils import make_rec
+
+def make_name(title):
+    return re.sub(r"[^a-zA-Z0-9]+", "-", title).lower()
+
+def upload_rec(rec):
+    result = get_action("package_create")({}, rec)
+    return result
+
+class ImportController(BaseController):
+    def index(self, group):
+        errors = request.params.get("errors", [])
+        if errors:
+            try:
+                errors = json.loads(errors)
+            except json.JSONDecodeError:
+                errors = ["Unknown errors detected"]
```

Comment:
This block is only reached on a JSONDecodeError, which would probably only happen if a user tried to manually visit a url like `/import?errors=not_valid_json`. I can change the error message to be something more descriptive perhaps, like "The 'errors' parameter is not valid JSON". This would probably be more accurate than the exception's message (something like "Invalid character at position 54: ,"). Thoughts?